cpp/dolfin/CMakeLists.txt

```diff
@@ -86,8 +86,15 @@ endif()
 # UFC
 target_include_directories(dolfin SYSTEM PUBLIC ${UFC_INCLUDE_DIRS})
 
+# Currently only used for Eigen
+set(DOLFIN_ALIGNMENT_BYTES "32" CACHE STRING "\
```

Comment:
Maybe `DOLFIN_EIGEN_MAX_ALIGNMENT_BYTES` is a better name?

Reply:
Sounds good.

Reply:
Ok